mistune/directives/toc.py

```diff
@@ -5,19 +5,21 @@
     The TOC directive syntax looks like::
 
         .. toc:: Title
+           :minlevel: 2
            :level: 3
 
-    "Title" and "level" option can be empty. "level" is an integer less
-    than 6, which defines the max heading level writers want to include
-    in TOC.
+    "Title", "minlevel", and "level" options can be empty. "minlevel" and
+    "level" are integers greater than or equal to 1 and less than 6, which
+    defines the min and max heading level writers want to include in TOC.
 """
 
 from .base import Directive, parse_options
 from ..toc import normalize_toc_item, render_toc_ul
 
 
 class DirectiveToc(Directive):
-    def __init__(self, level=3, heading_id=None):
+    def __init__(self, level=3, heading_id=None, minlevel=1):
```

Comment:
I made `minlevel` the last keyword argument to try and make this as non-breaking as possible, but it might make more sense to put it before `level`.